backend/tests/Designer.Tests/Services/OptionListReferenceServiceTests.cs

```diff
@@ -31,7 +31,38 @@ public async Task GetAllOptionListReferences_ShouldReturnAllReferences_WhenOptio
         Assert.Equivalent(optionListsReferences, expectedResponseList);
     }
 
-    private List<OptionListReference> OptionListReferenceTestData()
+    [Fact]
+    public async Task GetAllOptionListReferences_ShouldReturnAllReferences_WhenAppContainsASubform()
+    {
+        // Arrange
+        const string RepoName = "app-with-groups-and-task-navigation";
+        var optionListReferenceService = GetOptionListReferenceServiceForTest();
+        List<OptionListReference> expectedResponseList = [
+        new()
+        {
+           OptionListId = "yesNo",
+           OptionListIdSources = [
+               new OptionListIdSource {
+                   ComponentIds = ["brand"],
+                   LayoutName = "brand",
+                   LayoutSetId = "subform",
+                   TaskId = null,
+                   TaskType = null
+               }
+           ]
+        }];
```

Comment:
There is something strange with this code style. Missing indentations makes the code hard to read. This is what I would expect it to look like:
```suggestion
        List<OptionListReference> expectedResponseList = [
            new()
            {
                OptionListId = "yesNo",
                OptionListIdSources = [
                    new OptionListIdSource
                    {
                        ComponentIds = ["brand"],
                        LayoutName = "brand",
                        LayoutSetId = "subform",
                        TaskId = null,
                        TaskType = null
                    }
                ]
            }
        ];
```
I thought we already had automated jobs for fixing the code style in the backend, but it seems like we still have work to do on that subject.